Providers/SwissBitcoinPayProvider.cs

```diff
@@ -118,5 +134,21 @@ public override Task PostProcessPaymentAsync(PostProcessPaymentRequest postProce
             }
             return Task.CompletedTask;
         }
+
+        /// <summary>
+        /// When true user can reprocess payment from MyAccount > Orders > OrderDetail
+        /// </summary>
+        public override Task<bool> CanRePostProcessPaymentAsync(Order order)
+        {
+            if (order == null)
+                throw new ArgumentNullException("order");
+
+            if (order.PaymentStatus == PaymentStatus.Pending && (DateTime.UtcNow - order.CreatedOnUtc).TotalSeconds > 5)

```

Comment:
Why 5 seconds and not more?
The initial invoice is valid for 1 hour. Does the user has the ability to get back to the first invoice?

Reply:
It's a standard test in payment plugins in nopcommerce.
Probably to allow time for the transaction to be processed and avoid double spending